packages/react-components/src/list/src/ListItem.jsx

```diff
@@ -8,6 +8,10 @@ const propTypes = {
      * A list item can vary in size.
      */
     size: oneOf(["inherit"]),
+    /**
+     * Default classes
+     */
+    className: string,
```

Comment:
We don't document the "className" prop since it's implicit that every component support it.